genet/__init__.py

```diff
@@ -19,3 +21,5 @@
 from genet.schedule_elements import Route, Schedule, Service, Stop
 from genet.use.road_pricing import Toll
 from genet.utils import elevation, google_directions, graph_operations
+
+pyproj.network.set_network_enabled(False)
```

Comment:
It stops it from trying to connect to the network to download transformation gridfiles on-the-fly (see #213). If it does try, there is a bug with handling self-signed SSL certificates that will cause an error.

Reply:
This wasn't needed before because the PROJ environment variable associated with this is set to OFF when downloading from homebrew/apt, but is ON when downloading from conda-forge (because in the former it automatically downloads the `proj-data` dataset too while it isn't included in the latter)